Core/Net/AutoUpdate.cs

```diff
@@ -21,10 +21,8 @@ public class AutoUpdate
 
         private readonly Uri latestCKANReleaseApiUrl = new Uri("https://api.github.com/repos/KSP-CKAN/CKAN/releases/latest");
 
-        private readonly Uri latestUpdaterReleaseApiUrl = new Uri(
-            "https://api.github.com/repos/KSP-CKAN/CKAN-autoupdate/releases/latest");
+        private readonly Uri lastestUpdaterUrl = new Uri("https://ckan-travis.s3.amazonaws.com/AutoUpdater.exe");
```

Comment:
I would much prefer it if we could make this URL in our own ksp-ckan.org domain. @techman83, can you set up a cname entry (say "travis") pointing to the amazonaws.com address so we don't hard-code an external URL?

Reply:
Hmm, we lose https in the process of setting a custom domain. Unless we go to the effort of configuring up CloudFront.

Now, we should be aware; We pay for the transit from AWS and we don't currently have a budget for large amounts of downloading from our infrastructure.

Reply:
Hmm, in that case, maybe we should keep this download on github.com.

How can we get a cleanly built version of AutoUpdate.exe? I could put a new release on the CKAN-autoupdate repo now, which would also fix #2140, but I'd have to use my locally built file which may or may not be safe depending on mono versions, etc.

Reply:
I'm not super across the Autoupdater stuff, but there are ways to have travis release a build onto GitHub.

https://docs.travis-ci.com/user/deployment/releases/

Reply:
I believe that's already how CKAN's releases are done, but I'm thinking that adding another exe file to the main release download list would confuse users. "Do I need to download this AutoUpdate.exe? Will CKAN work without it?" Better to handle something like that in the background, as it is now.

Reply:
Possibly. We could setup a specific repo for it though, with a Readme stating its purpose.

Reply:
That sounds like the optimal situation. Revive CKAN-autoupdate repo and use it to push releases of the autoupdater, with a readme and release notes that make it clear this is not something that users should need to install manually. The release script could handle that without any problem?

Reply:
I think the release script can't do cross-repo uploads. This one runs on KSP-CKAN/CKAN and can handle releases in that repo, but not KSP-CKAN/CKAN-autoupdate. At least, I'm not able to find documentation or informal discussion of such.

Which implies either manually created releases, or reviving the _code_ of CKAN-autoupdate as well.

Reply:
Well, I think I'd rather have the Autoupdater included in the main repo releases than have a manual process

Reply:
OK, I'll look into that. Currently the auto-update code in ckan assumes the new exe is the first download in the list, so we would either need to control the ordering or add some smarter filtering so it can tell them apart.

I think I'm going to close this PR because we've pretty much decided against this approach, so any further update would pretty much blow these changes away and replace them. Might as well start fresh.